Makefile

```diff
@@ -7,8 +7,8 @@ INSTALL_BIN := install -D -m755
 
 # Julian Chesterfield, July 2007
 #
-# Copyright (c) 2007 XenSource, Inc. All use and distribution of this
-# copyrighted material is governed by and subject to terms and
+# Copyright (c) 2023 Cloud Software Group, Inc. All use and distribution
```

Comment:
Similar to discussed, this is history info, need not to change/update, you can add new item.